overlord/snapstate/conflict.go

```diff
@@ -243,3 +253,37 @@ func checkChangeConflictIgnoringOneChange(st *state.State, instanceName string,
 
 	return nil
 }
+
+// CheckSetKernelCmdlineConflict checks that no active change other
+// than ignoreChangeID has a task that touches the kernel command
+// line.
+func CheckSetKernelCmdlineConflict(st *state.State, ignoreChangeID string) error {
+	// check whether there are other changes that need to run exclusively
+	if err := checkChangeConflictExclusiveKinds(st, "", ignoreChangeID); err != nil {
+		return err
```

Comment:
this bit is not tested, for the usages in this PR is not strictly needed because doInstall always involve at least another more general conflict check but OTOH maybe it does make sense for the usage of this in the new?